.github/ISSUE_TEMPLATE/xtable_question.yml

```diff
@@ -0,0 +1,39 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+name: XTable Question
```

Comment:
Can we have questions just link to the discussions tab?

Reply:
Good idea!

Reply:
I was imagining something more like they do in Apache Doris: https://github.com/apache/doris/blob/master/.github/ISSUE_TEMPLATE/config.yml#L22

I'd prefer questions just go into a single place so the Issues section is about issues with the code and not general questions.

Reply:
Make sense.